### PR TITLE
docs: add beta documentation for Disable Resources feature

### PR DESCRIPTION
**Pull request by lizzthabet**
This PR adds documentation for the Disable Resources feature based on feedback that @lianmakesthings, @hyu, and @landism graciously gave! [Preview link is here.](https://deploy-preview-1025--eager-austin-142694.netlify.app/manual_update_control.html#beta-disabling-resources-through-tilt)

A couple thoughts:

I added the new content to the existing `Play and Pause Resources with Manual Update Control` page, since it relates to that content. I'm happy to stick it on its own page if that makes more sense.

Now that I'm seeing the docs on our site, it does feel a little clunky to have the technical information first, followed by the "why would you use this?" However, because this is a beta feature and the intention is to provide preliminary documentation for our beta users, I'm not too worried about telling a compelling story about this feature.

I also added a small `BETA` tag with pill tag styles. Happy to change this to something else, since it's a new design element and I didn't consult anyone about it. 🤓 

Closes [ch13120](https://app.shortcut.com/windmill/story/13120/document-mvp)

### CODE REVIEW

File: docs/manual_update_control.md [hunk 1]
```diff
@@ -57,3 +57,119 @@ To configure a resource to _only_ run when explicitly triggered from the UI, set
...
+Disabling a resource will stop the running process (if any) and delete any objects owned by that resource. It’s the equivalent of running `tilt down` on a specific resource. Enabling a resource will create, build, deploy, and start any processes for that resource.
+
+> 💡 Disabling and enabling a resource doesn’t take into account any of its resource dependencies. For example, if resource B depends on resource A, disabling resource A will not disable resource B.
+
+> 💡 If you’re using [`tilt args`](tiltfile_config.html) to define a subset of resources Tilt is running, only the resources that are defined through args will show up in the UI Dashboard. In a future release, we’re planning to display all defined resources from the Tiltfile in the UI.
+
+### How to enable the feature
+Disabling resources as a feature is turned off by default. To enable the feature and start using it, you’ll need to be on [Tilt v0.23.5+](https://github.com/tilt-dev/tilt/releases).
+
+Add the following line to your Tiltfile:
```
Comment (landism) [inline docs/manual_update_control.md:74]:
```suggestion
While in beta, the Disable Resources feature requires opting in. First, make sure you're on [Tilt v0.23.5+](https://github.com/tilt-dev/tilt/releases).

Then add the following line to your Tiltfile:
```

1. Make clear the "off by default" behavior is temporary
2. "disabling resources as a feature" is a really awkward noun phrase!
3. add "first," to indicate that there's more coming beyond being on a given tilt version
4. it just seems good to reduce usage of the word "enable" given its multiple meanings in this context

(FWIW, I only felt (1) was really worth opening a comment for, the rest are nits that you can feel free to drop)

**Review by landism — APPROVED** [review]
🙌

**Review by lianmakesthings — APPROVED** [review]
@SurbhiSGupta A "beta" tag has been implemented here. I think it'd be good if the tag we discussed yesterday would fit in here as well so we have the same iconography across our docs for feature state tagging
Screenshot 2022-01-07 at 14 41 26

File: docs/manual_update_control.md (hunk 1 above)
Comment (lizzthabet) [reply docs/manual_update_control.md:74]: sounds good! those changes make sense to me.

**Pull request merged**